src/pages/posts/2019/february/hooks-tips-instance-variables.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Hooks Tips Ep. 2 - Migrating Instance Variables
+publishDate: '02-22-2019'
+---
+
+import Footer from '@components/PostFooter.js'
+
+# Migrating Instance Variables
+
+> This blog post is part of a series on hooks, this blog post assumes you have a
+> decent initial understanding of React Hooks. I highly suggest starting with
+> the [ReactJS Docs](https://reactjs.org/docs/hooks-intro.html) to learn more.
+
+Frequently, `class` based components can have a decent number of Instance
+variables on them that are unrelated to state or props. These might be used to
+store refs for elements, or other data that you may need within the component.
```

Comment:
### Additional Notes:

Might be worth giving a quick reminder of what refs are good for - you almost mention it here. 

But something like, "Updating instance property values does not cause a re-render"

They also don't cause any lifecycle methods to fire when changed.